### DISCUSSION TITLE
Running a proxy server on top of turborepo

### DISCUSSION THREAD

**Discussion by vorillaz** [vercel/turborepo#185] [Ideas]
Hey there, turborepo looks amazing and I have been playing around with for the past few days, it looks amazing.

I was wondering if it's possible to run a proxy server of top of turborepo in order to connect all the apps over HTTP calls? 
That feature would be amazing as it will create a solid development experience.

It would be amazing if I could use turborepo or another tool on top of turborepo connecting all the internal services, eg:

```
-- proxy localhost:3000
--- api: localhost:3001 as localhost:3000/api
--- front:  localhost:3002 as localhost:3000/*
```

**Comment by anthonyshew** [comment]
Closing in favor of the more recent https://github.com/vercel/turbo/discussions/7462
